controller/notesapicontroller.php

```diff
@@ -125,23 +127,35 @@ public function create($content) {
      *
      * @param int $id
      * @param string $content
+     * @param int $modified
      * @param boolean $favorite
      * @return DataResponse
      */
-    public function update($id, $content=null, $favorite=null) {
+    public function update($id, $content=null, $modified=0, $favorite=null) {
```

Comment:
If `modified` is not stated in the request, the behavior is the same as before of my changes: the `mtime` of the note's file is set to the current date/time.

Therefore, my changes of the API are totally backward compatible. However, the API documentation has to be revised. But the wiki has not been transferred, yet.